NGitLab/IGroupsClient.cs

```diff
@@ -12,11 +14,15 @@ public interface IGroupsClient
 
         IEnumerable<Group> Search(string search);
 
+        GitLabCollectionResponse<Group> SearchAsync(string search);
```

Comment:
No CancellationToken?

Same for all other methods that return `GitLabCollectionResponse<T>`.

Reply:
Nope, there is no need for a `CancellationToken` here. You can set the `CancellationToken` when iterating on the collection (`await foreach(var item in collection.WithCancellation(token))`)

Reply:
Fair enough, but can you add documentation to that effect? When people see a method whose name ends with Async, they might expect it to take in a CancellationToken.

Reply:
Documentation added